src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -391,16 +391,24 @@ public IEnumerable<PropertyMirror> GetProperties()
 
                 ProcedureTable procedureTable = ClassNode.ProcTable;
                 List<ProcedureNode> procList = procedureTable.Procedures;
-                string getterPrefix = ProtoCore.DSASM.Constants.kGetterPrefix;
+                string getterPrefix = Constants.kGetterPrefix;
                 int prefixLength = getterPrefix.Length;
 
                 foreach (ProcedureNode pNode in procList)
                 {
                     name = pNode.Name;
                     bool hidden = pNode.MethodAttribute == null ? false : pNode.MethodAttribute.HiddenInLibrary;
-                    if (!hidden && name.Contains(getterPrefix) && pNode.ArgumentInfos.Count == 0)
+
+                    if (!hidden && name.Contains(getterPrefix))
                     {
-                        properties.Add(new PropertyMirror(pNode));
+                        // Auto-generated static property that takes the instance as the only input parameter.  
+                        var isAutoGeneratedThisProperty = pNode.IsAutoGeneratedThisProc && pNode.ArgumentInfos.Count == 1 &&
+                            pNode.ArgumentTypes[0].UID == pNode.ClassID;
```

Comment:
curious if this last check if required - I think it's good! - I just want to know what case is being filtered out?

Reply:
There could be methods beginning with a `get_` that may take arguments that do not match with the class type. This is to filter out those methods.